### DISCUSSION TITLE
When will the context be extendable?

### DISCUSSION THREAD

**Discussion by moloti** [ory/keto#1417] [Q&A]
I wanted to ask if there is a timeline for when the extended context will be released. So besides the subject context, other attributes can be added (tenant id, IP, device id, etc.).

The feature would help me a lot. Thank you!

**Comment by vinckr** [comment]
Hello @moloti 

There is no timeline for extended context yet, as far as I can see there is also no design document/detailed feature request yet. 
I recommend subscribing to the [Ory Changelog](https://changelog.ory.sh/) or the Ory Keto open issues & pull requests for updates on this feature.
If you want to speed up delivery of this feature, feel free to [contact the team](https://www.ory.sh/contact/) to discuss options.

**Comment by alaghusun60** [comment]
which one correct version for ory keto . we use v0.6.0-alpha.1. We use keto -c ./config/keto.yml migrate up --yes for psql migrate. We got only three table get  keto_relation_tables , networks,schema_migeration. Other tables not found. Wether this enough
